src/contentmanager.cpp

```diff
@@ -213,6 +213,15 @@ void ContentManager::eraseBookFilesFromComputer(const QString fileToRemove)
 
 void ContentManager::eraseBook(const QString& id)
 {
+    auto tabBar = KiwixApp::instance()->getTabWidget();
+    for (int i = 1; i < tabBar->count() - 1; i++)
+    {
+        WebView* webView = tabBar->widget(i);
+        if (webView->zimId() == id) {
+            tabBar->closeTab(i);
+            i--;
```

Comment:
Hum, I understand why you decrement `i` (the `tabBar` count change when you remove a tab).
But I would prefer a while here (instead of decrement and increment).
```
int i = 1;
while (i < tasBar->count() - 1) {
  WebView* webView = tabBar->widget(i);
  if (webView->zimId() == id) {
    tabBar->closeTab(i);
  } else {
    i++;
  }
}
```